specification/synapse/resource-manager/readme.md

```diff
@@ -28,16 +28,39 @@ These are the global settings for the Azure Synapse Analytics API.
 description: Azure Synapse Analytics Management Client
 openapi-type: arm
 azure-arm: true
-tag: package-2021-03
+tag: package-preview-2021-04
 generate-empty-classes: true
+modelerfour:
+  lenient-model-deduplication: true
 ```
 
 
+### Tag: package-preview-2021-04
+
+These settings apply only when `--tag=package-preview-2021-04` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2021-04'
+input-file:
+  - Microsoft.Synapse/preview/2021-04-01-preview/bigDataPool.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/checkNameAvailability.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/firewallRule.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/integrationRuntime.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/keys.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/library.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/operations.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privateEndpointConnections.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privateLinkResources.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privatelinkhub.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/sqlPool.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/sqlServer.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/workspace.json
+```
 ### Tag: package-2021-03
```

Comment:
Nice to add a space between this two lines

Reply:
Line added